source/_integrations/librehardwaremonitor.markdown

```diff
@@ -0,0 +1,78 @@
+---
+title: Libre Hardware Monitor
+description: Instructions on how to integrate Libre Hardware Monitor within Home Assistant.
+ha_category:
+  - System monitor
+ha_release: 2025.4
+ha_iot_class: Local Polling
+ha_domain: librehardwaremonitor
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+ha_quality_scale: bronze
+---
+
+The `librehardwaremonitor` {% term integration %} uses your [Libre Hardware Monitor](https://github.com/LibreHardwareMonitor/LibreHardwareMonitor) installation as a source for sensors that will display system information.
+
+## Prerequisites
+
+Libre Hardware Monitor must be running on the host, with "Remote web server" active. You also need to open inbound port (TCP 8085) on the host.
+
+To open port (on Windows):
+
+1. Navigate to Control Panel, System and Security and Windows Firewall.
+2. Select **Advanced settings** and highlight **Inbound Rules** in the left pane.
+3. Right click Inbound Rules and select New Rule.
+4. Add the port you need to open and click Next.
```

Comment:
```suggestion
4. Add the port you need to open and select **Next**.
```
In general, we avoid [click and use select instead](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/click)